mlir/test/Conversion/MemRefToEmitC/memref-to-emitc-failed.mlir

```diff
@@ -46,14 +46,6 @@ memref.global "nested" constant @nested_global : memref<3x7xf32>
 
 // -----
 
-func.func @unsupported_type_f16() {
-  // expected-error@+1 {{failed to legalize operation 'memref.alloca'}}
-  %0 = memref.alloca() : memref<4xf16>
```

Comment:
It could be nice to keep this test as well and replace it with an unsupported floating point type as suggested by @simon-camp for the arith to emitc tests.